virttest/standalone_test.py

```diff
@@ -27,8 +27,8 @@ def _variant_only_file(filename):
 SUPPORTED_LIBVIRT_URIS = ['qemu:///system', 'lxc:///']
 SUPPORTED_LIBVIRT_DRIVERS = ['qemu', 'lxc', 'xen']
 
-SUPPORTED_IMAGE_TYPES = ['raw', 'qcow2', 'qed', 'vmdk']
-SUPPORTED_DISK_BUSES = ['ide', 'scsi', 'virtio_blk',
+SUPPORTED_IMAGE_TYPES = ['qcow2', 'raw', 'qed', 'vmdk']
+SUPPORTED_DISK_BUSES = ['virtio_blk', 'ide', 'scsi',
                         'virtio_scsi', 'lsi_scsi', 'ahci', 'usb2', 'xenblk']
```

Comment:
What is the reason behind this reordering? It doesnt't seem alphabetical so I guess for some other purpose?

Reply:
The goal is to provide defaults such as here:

* https://github.com/avocado-framework/avocado-vt/pull/2623/files#diff-f3d8214ff094a0391aa124bd9e6a5cbcR129
* https://github.com/avocado-framework/avocado-vt/pull/2623/files#diff-f3d8214ff094a0391aa124bd9e6a5cbcR134
* https://github.com/avocado-framework/avocado-vt/pull/2623/files#diff-f3d8214ff094a0391aa124bd9e6a5cbcR143

And a convention so that the first entry is the default.  But, maybe it's a very bad idea to *not* document that in those entries.  I mean:

```
#: First entry of this list will be the default disk image
SUPPORTED_DISK_BUSES = [ ... ]
```

Reply:
Ah I see, yes an extra line like this would be great to explain everything in a compact manner!

Reply:
And I other places where the same thing could be used, like here:

https://github.com/clebergnu/avocado-vt/blob/new_settings/avocado_vt/plugins/vt.py#L56

I'm adding those too in this commit (and adding a pointer to the docstring/comment).